source/admin_manual/list_of_events.rst

```diff
@@ -1113,6 +1113,29 @@ IMAP Command
    .. Note:: This event is currently not sent for pre-login IMAP commands.
 
 
+.. dovecot_core:event:: imap_id_received
+   :added: v2.4.0;v3.0.0
+   :inherit: imap_client
+
+   :field cmd_name: Name of the command.
```

Comment:
I don't think this event currently has cmd_name field, at least I don't see it in the code. But if it was inheriting from imap_command, it would inherit it from there (for post-login).